### ISSUE TITLE
circlescan generate floats

### ISSUE THREAD

**Issue by gszathmari** [dpmcmlxxvi/pixelscan#6]
The example in README.rst in not correct

My code is:

```
from pixelscan.pixelscan import *

for x, y in circlescan(0, 0, 0, 1):
   print (x, y)
```

What I expect:

```
(0,0), (0,1), (1,1), (1,0), (1,-1), (0,-1), (-1,-1), (-1,0), (-1,1)
```

What I got:

```
(0, 0)
(0, 1)
(0.707107, 0.707107)
(1, 0)
(0.707107, -0.707107)
(0, -1)
(-0.707107, -0.707107)
(-1, 0)
(-0.707107, 0.707107)
```

**Comment by gszathmari** [comment]
My python version is:
`Python 2.7.6 (default, Jun 22 2015, 17:58:13)`

My pixelscan version is `pixelscan==0.3.2`

**Comment by dpmcmlxxvi** [comment]
There was a snap transformation missing in the README example. Thanks. It's been fixed with aa67c30.
